helpdesk_mgmt/data/helpdesk_data.xml

```diff
@@ -45,7 +45,7 @@
                           <table width="590" border="0" cellpadding="0" bgcolor="#ffffff" style="min-width: 590px; background-color: rgb(255, 255, 255); padding: 20px; border-collapse:separate;">
                             <tbody>
                               <td valign="top" style="font-family:Arial,Helvetica,sans-serif; color: #555; font-size: 14px;">
-                                <p>Hello ${object.user_id.name},</p>
+                                <p>Hello ${object.user_id.name and object.user_id.name or ''|safe},</p>
```

Comment:
`safe` part is for emails.

```suggestion
                                <p>Hello ${object.user_id.name and object.user_id.name or ''},</p>
```

Now that we are here, the user is the assigned responsible, but this mail is sent to the customer (object.partner_id), so it's not coherent.

I think we should put the partner's name.